pkg/network/ebpf/c/protocols/protocol-classification-helpers.h

```diff
@@ -75,7 +91,7 @@ static __always_inline void classify_protocol(protocol_t *protocol, const char *
 
     if (is_http(buf, size)) {
         *protocol = PROTOCOL_HTTP;
-    } else if (is_http2(buf, size)) {
+    } else if (is_http2(buf, size) || is_http2_server_settings(buf, size)) {
```

Comment:
Consider moving the call to `is_http2_server_settings` to keep this function cleaner

Reply:
changed